File: app/src/main/java/agi/erecreditsmanager/ForLecture/ForLectureLayout.java
package agi.erecreditsmanager.ForLecture;

import android.content.Context;
import android.view.LayoutInflater;
import android.widget.Button;
import android.widget.RelativeLayout;
import android.widget.TextView;

import agi.erecreditsmanager.R;

public class ForLectureLayout extends RelativeLayout {

    Context context;
    TextView forLectureTextView;
    Button deleteButton;

    public ForLectureLayout(Context context) {
        super(context);
        this.context = context;

        init();
    }

    private void init() {
        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        inflater.inflate(R.layout.forlecture, this, true);

        forLectureTextView = (TextView) findViewById(R.id.forLectureTextView);
        deleteButton = (Button) findViewById(R.id.deleteButton);
    }

    public void setForLectureTextView(String text) {
        forLectureTextView.setText(text);
    }

    public TextView getForLectureTextView() {
        return forLectureTextView;
    }

    public Button getDeleteButton() {
        return deleteButton;
    }
}

File: app/src/main/java/agi/erecreditsmanager/ForLecture/ForLectureLayout.kt
package agi.erecreditsmanager.ForLecture

import agi.erecreditsmanager.R
import android.content.Context
import android.view.LayoutInflater
import android.widget.Button
import android.widget.RelativeLayout
import kotlinx.android.synthetic.main.forlecture.view.*

class ForLectureLayout(context : Context) : RelativeLayout(context) {
    init {
        LayoutInflater.from(context).run {
            inflate(R.layout.forlecture, this@ForLectureLayout, true)

            forLectureTextView
            deleteButton
        }
    }

    //MainAdapter 재구현 후 삭제
    fun setForLectureTextView(text : String) {
        forLectureTextView.text = text
    }

    fun getDeleteButton() : Button {
        return deleteButton
    }
}